modules/exploits/linux/http/grandstream_ucm62xx_sendemail_rce.rb

```diff
@@ -0,0 +1,176 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+  Rank = ExcellentRanking
+
+  prepend Msf::Exploit::Remote::AutoCheck
+  include Msf::Exploit::Remote::HttpClient
+  include Msf::Exploit::CmdStager
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Grandstream UCM62xx IP PBX sendPasswordEmail RCE',
+        'Description' => %q{
+          This module exploits unauthenticated SQL and command injection vulnerabilities affecting
+          the Grandstream UCM62xx IP PBX series of devices. The vulnerabilities allow an
+          unauthenticated and remote attacker to execute commands as root. This module affects
+          UCM62xx versions before firmware version 1.0.19.20.
```

Comment:
See above comment about how this is chained together. Think this description could explain the connection between these vulns and the affected pages + parameters a bit more.